StripePaymentSheet/StripePaymentSheet/Source/Analytics/PaymentSheetAnalyticsHelper.swift

```diff
@@ -148,6 +176,13 @@ final class PaymentSheetAnalyticsHelper {
                 case .link:
                     return (.mcOptionSelectCompleteLink, nil)
                 }
+            case .embedded:
+                if case .saved(let paymentMethod) = option {
+                    return (.mcOptionSelectEmbeddedSavedPM, paymentMethod.type.identifier)
+                } else {
+                    stpAssertionFailure("Embedded should only use this function to record tapped saved payment methods")
+                    return (.unexpectedErrorPaymentSheetAnalytics, nil)
```

Comment:
I think we need to send more details in this, otherwise we won't know what the unexpected error was. 

But I'm also okay to only assert and not send an analytic.

Reply:
Removed analytic